src/test/ui-fulldeps/proc-macro/derive-helper-shadowing.stderr

```diff
@@ -1,19 +1,20 @@
-error[E0659]: `my_attr` is ambiguous
+error[E0659]: `my_attr` is ambiguous (derive helper attribute vs any other name)
   --> $DIR/derive-helper-shadowing.rs:6:3
    |
 LL | #[my_attr] //~ ERROR `my_attr` is ambiguous
    |   ^^^^^^^ ambiguous name
    |
-note: `my_attr` could refer to the name defined here
+note: `my_attr` could refer to the derive helper attribute defined here
   --> $DIR/derive-helper-shadowing.rs:7:10
    |
 LL | #[derive(MyTrait)]
    |          ^^^^^^^
-note: `my_attr` could also refer to the name imported here
+note: `my_attr` could also refer to the attribute macro imported here
   --> $DIR/derive-helper-shadowing.rs:4:5
    |
 LL | use derive_helper_shadowing::*;
    |     ^^^^^^^^^^^^^^^^^^^^^^^^^^
+   = help: use `self::my_attr` to refer to this attribute macro unambiguously
```

Comment:
Same answer as https://github.com/rust-lang/rust/pull/55884#discussion_r232983297.

Derive helper is a high-priority thing though, so it will be automatically preferred if the "name vs any name in imports" ambiguity errors (which are technically unnecessary) are relaxed, and "derive helper vs any any name anywhere" ambiguity erros (which needs some preliminary bugfixing done) are relaxed as well.